docs/app/docs/page/blocks/basic-blocks/ngd-overview-block.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'ngd-overview-block',
+  template: `
+    <div class="block-container">
+      <h2 class="class-name">
+        <a [routerLink]="" fragment="{{source.name}}" ngdFragment></a>
```

Comment:
shouldn't we use spaces around property name inside of `{{ }}`?

Reply:
done